modules/web/src/app/project/component.ts

```diff
@@ -232,16 +232,25 @@ export class ProjectComponent implements OnInit, OnChanges, OnDestroy {
     if (!this.isEnterpriseEdition) {
       return;
     }
-
-    for (const project of projects) {
-      const quota$ = this.currentUser.isAdmin
-        ? this._quotaService.quotas.pipe(map(quotas => quotas.find(({subjectName}) => subjectName === project.id)))
-        : this._quotaService.getLiveProjectQuota(project.id);
-
-      quota$.pipe(filter(Boolean), takeUntil(this._unsubscribe)).subscribe(_ => {
-        this.hasQuota = true;
-        this._cdr.detectChanges();
-      });
+    if (this.currentUser?.isAdmin) {
+      this._quotaService.quotas
+        .pipe(take(1))
+        .pipe(filter(quotas => !!quotas?.find(quotaDetails => !_.isEmpty(quotaDetails?.quota))))
+        .subscribe(_ => {
+          this.hasQuota = true;
+          this._cdr.detectChanges();
+        });
+    } else {
+      for (const project of projects) {
```

Comment:
We should check if the project is active otherwise there is no point in trying to retrieve the quotas. At least let's do it here since we are already fixing things.